pype/plugins/ftrack/publish/collect_ftrack_family.py

```diff
@@ -0,0 +1,60 @@
+"""
+Requires:
+    none
+
+Provides:
+    instance     -> families ([])
+"""
+import os
+import pyblish.api
+
+from pype.lib.plugin_tools import filter_profiles
+
+
+class CollectFtrackFamily(pyblish.api.InstancePlugin):
+    """
+        Adds explicitly 'ftrack' to families to upload instance to FTrack.
+
+        Uses selection by combination of hosts/families/tasks names via
+        profiles resolution.
+
+        Triggered everywhere, checks instance against configured
+    """
+    label = "Collect Ftrack Family"
+    order = pyblish.api.CollectorOrder + 0.4999
+
+    profiles = {
+        "hosts": ["standalonepublisher"],
+        "families": [],
+        "task_names": [],
+        "add_to_ftrack": True
+    }
+
+    def process(self, instance):
+        if not self.profiles:
+            self.log.warning("No profiles present for adding Ftrack family")
+            return
+
+        anatomy_data = instance.context.data["anatomyData"]
+        task_name = instance.data("task",
+                                  instance.context.data["task"])
+        host_name = anatomy_data.get("app",
```

Comment:
I would use only `os.environ["AVALON_APP"]` here. Anatomy data can change and may not reffer to host name. Env `AVALON_APP` is dedicated to host name.

Reply:
There should be probably some API method to get this combination (at least in OpenPype) as this is kinda ridiculous. (Or confusing at least.) :)

Reply:
Like:
```
def get_host_name():
    return os.environ.get("AVALON_APP")
```

Reply:
I'll fix it here, but keep it original way in OpenPype. If we want venture more into automatic testing, checking env vars is PITA, imho. There is no reason why host name shouldn't be on instance or context, again imho.